qa/standalone/misc/test-mclock-profile-switch.sh

```diff
@@ -190,6 +190,43 @@ function TEST_profile_custom_to_builtin() {
     teardown $dir || return 1
 }
 
+function TEST_profile_disallow_builtin_params_modify() {
+    local dir=$1
+
+    setup $dir || return 1
+    run_mon $dir a || return 1
+    run_mgr $dir x || return 1
+    run_osd $dir 0 --osd_op_queue=mclock_scheduler || return 1
+
+    # Verify that the default mclock profile is set on the OSD
+    local mclock_profile=$(ceph config get osd.0 osd_mclock_profile)
+    test "$mclock_profile" = "high_client_ops" || return 1
+
+    # Try and change a mclock config param and confirm that no change occurred
+    local client_res=$(CEPH_ARGS='' ceph --format=json daemon $(get_asok_path \
+      osd.0) config get osd_mclock_scheduler_client_res | \
+      jq .osd_mclock_scheduler_client_res | bc)
+    local client_res_new=$(expr $client_res + 10)
+    ceph config set osd.0 osd_mclock_scheduler_client_res \
+      $client_res_new || return 1
+    sleep 2 # Allow time for changes to take effect
+
+    # Check configuration value on Mon store for the osd
+    local res=$(ceph config get osd.0 \
+      osd_mclock_scheduler_client_res) || return 1
+    echo "Mon db: osd.0 osd_mclock_scheduler_client_res = $res"
+    test $res -ne $client_res_new || return 1
+
+    # Check value in the in-memory 'values' map is unmodified
+    res=$(CEPH_ARGS='' ceph --format=json daemon $(get_asok_path \
+      osd.0) config get osd_mclock_scheduler_client_res | \
+      jq .osd_mclock_scheduler_client_res | bc)
+    echo "Values map: osd.0 osd_mclock_scheduler_client_res = $res"
+    test $res -ne $client_res_new || return 1
+
+    teardown $dir || return 1
+}
+
```

Comment:
Should there be a test case for "config show" command as well?

Reply:
Added another check to show the configuration for the osd.